documentation/api/io.strimzi.api.kafka.model.CruiseControlSpec.adoc

```diff
@@ -1,23 +1,43 @@
-// Module included in the following assemblies:
-//
-// assembly-cruise-control-concepts.adoc
+Configures a Cruise Control cluster.
 
-[id='ref-cruise-control-configuration-{context}']
-= Cruise Control configuration
+Configuration options relate to:
 
-The `config` property in `Kafka.spec.cruiseControl` contains configuration options as keys with values as one of the following JSON types:
+* Goals configuration
+* Capacity limits for resource distribution goals
+
+[id='property-cruise-control-config-{context}']
+=== `config`
```

Comment:
Is this a good title? Just the field in the CR?

Reply:
We use this approach throughout the API reference: Intro to schema, then description + examples of specific properties where needed.  I guess we could take another approach to title, though I think the heading levels should be sufficient to understand grouping.

Reply:
Ok ok fine with me.